pandas/tests/resample/test_base.py

```diff
@@ -199,7 +200,15 @@ def test_resample_empty_dataframe(empty_frame_dti, freq, resample_method):
 
     rs = df.resample(freq, group_keys=False)
     result = getattr(rs, resample_method)()
-    if resample_method != "size":
+    if resample_method == "ohlc":
+        # TODO: no tests with len(df.columns) > 0
```

Comment:
Can these be added here?

Reply:
im falling behind on things.  mind if i punt on this?

Reply:
Yep - sounds good.